content/articles/billing-settings.markdown

```diff
@@ -14,32 +14,29 @@ categories:
 
 ---
 
-
-## Invoices without custom billing information
-
-Until we are provided with further information our invoices will only reflect the email address of the DNSimple account the invoice was sent to. All invoices will be sent to your DNSimple account's email address.
+Until we are provided with further information our invoices will only reflect the email address of the DNSimple account that you signed up with. And all invoices will be sent to that email address.
```

Comment:
I'm not sure I would rephrase this section. Since multi-accounts, this sentence seems to indicate that it will be sent to the email you signed with. But if your user has 4 accounts, each account has its own email and that's where the invoice will be sent.